test/integration/advertisements_test.rb

```diff
@@ -5,7 +5,7 @@ class AdvertisementsTest < ActionDispatch::IntegrationTest
   setup do
     Rails.cache.clear
     start_date = Date.parse("2019-01-01")
-    @premium_campaign = campaigns(:premium_bundled)
+    @premium_campaign = amend(campaigns: :premium_bundled, start_date: start_date, end_date: start_date.advance(months: 3))
```

Comment:
Scared to merge this bc im not sure what you were testing here. I assume it may just be that the start and end date were correctly syncing from the bundle to campaign when the campaign is saved but I want to make sure.

Since the bundled campaign is no longer syncing its start and end dates with the campaign bundle, the campaign we create here, the properties can't find it bc it's in the future from where they think time is currently.

Reply:
These tests were written before the system tests and are just verifying that the controller emits the expected output. Feel free to tweak test setup as needed for this new behavior on bundled campaigns.

Reply:
ok then this should be good to merge.